css/main.css

```diff
@@ -25,6 +39,12 @@ html {
 
 .nav-menu-btn {
   background-color: transparent;
+  color: #495057;
+  transition: all 0.5s;
+}
+
+.onscroll-color {
+  color: #f5f5f5;

```

Comment:
[Suggestion] You can CSS variable here which will be a good approach to reuse properties. 

Reference:  https://www.w3schools.com/css/css3_variables.asp

Reply:
Thanks for pointing that out,actually it didn't pop out in my mind
will try it next time

Reply:
Cool

Reply:
Testing comment. Ignore it